src/aat/resources/features/F-051/F-051.feature

```diff
@@ -16,13 +16,12 @@ Background:
 
 
   @S-109
-  Scenario: must return appropriate negative response for a user not having a profile in CCD
+  Scenario: must return appropriate user setting from idam role for a user not having a profile in CCD
     Given a user with [no profile in CCD]
     When a request is prepared with appropriate values
-    And the request [uses a Case ID that doesn’t exist in CCD]
     And it is submitted to call the [Get default settings for user] operation of [CCD Data Store]
-    Then a negative response is received
-    And the response [has the 403 return code]
+    Then a positive response is received
+    And the response [has the 200 return code]
```

Comment:
This is a potential breaking change and should be checked if there is any consumer of this operation relying on the previous behaviour.

Reply:
The line 24 can be removed, since, from a behavioural specification point of view, "Then a positive response is received" would be sufficient.

Reply:
will do, someone may need to retrospectively go through all current scenarios as those 2 lines are seen together throughout the test suite

Reply:
you're right in that.